### DISCUSSION TITLE
Adding Properties/Object to Controls on runtime.

### DISCUSSION THREAD

**Discussion by anilgenius** [dotnet/csharplang#8385] [Language Ideas]
In VFP we can Add properties at runtime to textbox/combo/grid/forms. Currently, C# is not supporting. Thats feature helps developer to make more rapid development in vfp. I  wish if that's available to C# it will cut of development time by half.  in VFP the syntax was
Textbox1.Addproperty, form1.Addproperty, Grid1.Addproperty as so on.

**Comment by bernd5** [comment]
Can you provide more details what you want?
